src/modules/CreateDrops/TokenDrop/CreateTokenDropContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropsWithChildren } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import * as z from 'zod'; // not sure why its not picking up from 'zod'
+
+import { WALLET_TOKENS } from './data';
+
+const schema = z.object({
+  dropName: z.string().min(1, 'Drop name required'),
+  selectedFromWallet: z.object({
+    symbol: z.string(),
+    amount: z.string(),
+  }),
+  selectedToWallets: z.array(z.string().min(1)).min(1, 'At least one wallet is required'),
+  totalLinks: z
+    .number({ invalid_type_error: 'Number of links required' })
+    .positive()
+    .min(1, 'Required'),
+  amountPerLink: z.number({ invalid_type_error: 'Amount required' }).gt(0),
+  redirectLink: z.union([z.string().url(), z.string().length(0)]).optional(),
```

Comment:
I think the `zod`url checker is a bit too strict. In the form, when I type `wallet.near.org/linkdrop`, it marks this as invalid since i'm not including `https://`.

Reply:
Alright, let me figure out a way to do that. Maybe we'd have to use a regex to validate